packages/reporters/bundle-analyzer/src/BundleAnalyzerReporter.js

```diff
@@ -142,7 +142,7 @@ async function getBundleNode(bundle: PackagedBundle, options: PluginOptions) {
   }
 
   return {
-    label: bundle.filePath,
+    label: relativePath,
```

Comment:
Is this what you had in mind?
```suggestion
    label: path.relative(options.projectRoot, bundle.filePath),
```

Reply:
@mischnic yep thanks!